include/amici/rdata.h

```diff
@@ -265,6 +265,12 @@ class ReturnData {
      */
     std::vector<int> preeq_numlinsteps;
 
+    /**
+     * number of simulation steps for adjoint steady state problem
+     * (preequilibration) [== 0 if analytical solution worked, > 0 otherwise]
+     */
+    int preeq_numstepsB = 0;
```

Comment:
We get an additional readout here, actually maybe even two: The time point, when steadystate was reached in backward integration could also be outputted, or even the backward steadystate (however: no clue what this would be good for atm): For models with singular Jacobian, the steadystate of xB does not need to be 0, but can be just some vector in the left-kernel of J.

Reply:
We could also think of doing something like a `backward_status`, since forward, we have a 3-vector for this...